dojo/unittests/tools/test_securecodebox_parser.py

```diff
@@ -47,6 +47,9 @@ def test_scb_parser_with_many_vuln_has_many_findings(self):
         self.assertEqual(findings[0].unsaved_endpoints[0].query, None)
         self.assertEqual(findings[0].unsaved_endpoints[0].fragment, None)
         self.assertEqual(findings[0].unsaved_endpoints[0].port, 22)
+        self.assertEqual(findings[5].unsaved_endpoints[0].path, "/robots.txt")
+        self.assertEqual(findings[5].unsaved_endpoints[0].fragment, "test")
+        self.assertEqual(findings[5].unsaved_endpoints[0].query, None)
```

Comment:
FYI: this form is also possible
```suggestion
        self.assertIsNone(findings[5].unsaved_endpoints[0].query)
```